docs-2.0/nebula-dashboard-ent/3.create-import-dashboard/2.import-cluster.md

```diff
@@ -9,6 +9,10 @@ This topic introduces how to import clusters using Dashboard. The current versio
     - In the same cluster, the service versions need to be unified. Importing Nebula Graph examples from different versions in the same cluster is not supported.
     - The installation path of Nebula Graph needs to be the default installation path `/usr/local`.
 
+1. In the configuration file of each service, change the IP in `<meta|graph|storage>_server_addrs` and `local_ip` to the server's IP, and then start Nebula Graph.
```

Comment:
I recommend changing configuration file to configuration files.
"each service" usually refers to the Graph Service, the Meta Service, and the Storage Service, but each of them may have more than one server, and that means there may be multiple configuration files.

Reply:
```suggestion
1. In the configuration files of each service, change the IP in `<meta|graph|storage>_server_addrs` and `local_ip` to the server's IP, and then start Nebula Graph.
```

Reply:
> 

got it
Thank you!